bower.json

```diff
@@ -26,7 +26,5 @@
   ],
   "dependencies": {
     "jquery": ">=1.0.0"
-  },
-  "devDependencies": {
```

Comment:
I remove dev dependencies from bower because it should not be used for dev on this project.